rust_src/src/multibyte.rs

```diff
@@ -138,6 +139,24 @@ fn raw_byte_from_codepoint(cp: Codepoint) -> c_uchar {
     (cp - 0x3F_FF00) as c_uchar
 }
 
+/// UNIBYTE_TO_CHAR macro
+#[inline]
+pub fn unibyte_to_char(cp: Codepoint) -> Codepoint {
+    if (cp as u8).is_ascii() {
```

Comment:
This cast is dangerous - it will truncate the codepoint so that thinks that aren't ascii will look like ascii.